vividus-extension-selenium/src/main/java/org/vividus/selenium/SauceLabsCapabilityType.java

```diff
@@ -28,4 +28,5 @@
     String DEVICE = "device";
     String CAPABILITIES = "capabilities";
     String SELENIUM_VERSION = "seleniumVersion";
+    String APP = "app";
```

Comment:
I think we'll remove `SauceLabsCapabilityType.java` at some point, so I don't recommend to keep using it